Cabal-syntax/src/Distribution/PackageDescription/FieldGrammar.hs

```diff
@@ -621,6 +621,8 @@ buildInfoFieldGrammar =
     <*> monoidalFieldAla "cc-options" (alaList' NoCommaFSep Token') L.ccOptions
     <*> monoidalFieldAla "cxx-options" (alaList' NoCommaFSep Token') L.cxxOptions
       ^^^ availableSince CabalSpecV2_2 []
+    <*> monoidalFieldAla "jsp-options" (alaList' NoCommaFSep Token') L.jspOptions
+      ^^^ availableSince CabalSpecV3_14 []
```

Comment:
Is it correct? I'd expect the next major release, so `CabalSpecV3_16`.

Reply:
@ulysses4ever  did i do it right?

Reply:
It needs to be `CabalSpecV3_16`, and that needs to be added if it doesn't exist yet. It is too late to add things to the 3.14 spec.

Reply:
I'm talking about new changes :)